persistence/src/main/java/com/epam/esm/specification/util/CertificateParamsRequestUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.esm.specification.util;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class CertificateParamsRequestUtil {
```

Comment:
Please refactor this class. Use a collection of all available sorting fields. You should check incoming sortBy against this collection and if the parameter is invalid do default sorting, throw an exception, or whatever.

Reply:
Created constants with values instead of collection